src/element-events.js

```diff
@@ -48,42 +49,25 @@ export class ElementEvents {
 
   /**
    * Adds and Event Listener on the context element.
-   * @param eventName
-   * @param handler
-   * @param bubbles
    * @return Returns the eventHandler containing a dispose method
    */
-  subscribe(eventName: string, handler: Function, bubbles?: boolean = true): EventHandler {
-    if (handler && typeof handler === 'function') {
-      handler.eventName = eventName;
-      handler.handler = handler;
-      handler.bubbles = bubbles;
-      handler.dispose = () => {
-        this.element.removeEventListener(eventName, handler, bubbles);
-        this._dequeueHandler(handler);
-      };
-      this.element.addEventListener(eventName, handler, bubbles);
-      this._enqueueHandler(handler);
-      return handler;
+  subscribe(eventName: string, handler: Function, captureOrOptions?: boolean | AddEventListenerOptions = true): EventHandler {
```

Comment:
This looks like a breaking change to the third parameter. Now the default would be the opposite of what it was before, correct?

Reply:
From the naming of the parameter, yes. From the code, no. It would be considered a bug from the naming imo

Reply:
That sounds good then.